autoload/fzf_tags.vim

```diff
@@ -52,10 +65,14 @@ function! s:tag_to_string(index, tag_dict)
     call add(components, s:magenta(a:tag_dict['filename']))
   endif
   if has_key(a:tag_dict, 'class')
-    call add(components, s:green(a:tag_dict['class']))
+    "call add(components, s:green(a:tag_dict['class']))
```

Comment:
These components are added to each line in order to filter on them with normal fzf syntax.